python_examples/rebound.py

```diff
@@ -1,4 +1,9 @@
 from ctypes import *
+import numpy
+import sys
```

Comment:
I think it would be better to make it forwards compatible, as follows:

    try:
        range = xrange  # this means python 2.x
    except NameError:
        pass # this means python >2.x (swallow exception)